### DISCUSSION TITLE
Database upgrade v15 / Android Version

### DISCUSSION THREAD

**Discussion by colourqueen** [moneymanagerex/moneymanagerex#5264] [Q&A] [answered]
I was keen to upgrade my desktop Money Manager to v 16 but took notice of the clear warning about the database not being likely to run with the Android version of MM. (I can confirm it does not!) I am also running an even older version of Android (2018 11 15 982) to allow sync with OneDrive and the desktop version. I know that the Android version is not being maintained due to lack of developers but I wondered if there was any way of making it work with the new database?  Can anyone suggest anything?

**Comment by tactilis** [comment, marked as answer]
Please see the discussion here:

[url]https://github.com/moneymanagerex/moneymanagerex/issues/5230[/url]

and here:

[url]https://github.com/moneymanagerex/moneymanagerex/issues/5245[/url]

The MMEX desktop app will continue to evolve and there will be further database changes in future releases.

Your only option is to stay on the version of MMEX (1.5.20/1.5.21) that is compatible with the out of date Android database.
